app/Helpers/MiscHelper.php

```diff
@@ -2,20 +2,20 @@
 
 namespace App\Helpers;
 
-class MiscHelper{
+class MiscHelper {
 
-    public static function array_diff_assoc_recursive($array1, $array2){
-        $difference=array();
-        foreach($array1 as $key => $value) {
-            if( is_array($value) ) {
-                if( !isset($array2[$key]) || !is_array($array2[$key]) ) {
+    public static function array_diff_assoc_recursive($array1, $array2) {
+        $difference = array();
+        foreach ($array1 as $key => $value) {
+            if (is_array($value)) {
+                if (!isset($array2[$key]) || !is_array($array2[$key])) {
                     $difference[$key] = $value;
-                } else {
+                }else {
                     $new_diff = MiscHelper::array_diff_assoc_recursive($value, $array2[$key]);
-                    if( !empty($new_diff) )
+                    if (!empty($new_diff))
```

Comment:
__[code_sniffer]__
* Inline control structures are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->